core/trino-main/src/test/java/io/trino/connector/MockConnector.java

```diff
@@ -576,6 +576,9 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
         @Override
         public void dropColumn(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle column) {}
 
+        @Override
+        public void dropField(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle column, List<String> field) {}
```

Comment:
throwing unsupported would be more appropriate...
why do we need this override at all?

(i know it's consistent with pre-existing dropColumn above, but i still don't udnerstand why we're doing this)

Reply:
No strong reason except for consistency with the existing `dropColumn`. Let me remove.